Annotation/declarative_golden_set.json

```diff
@@ -640,7 +701,7 @@
                     }
                 ],
                 "type": "declarative",
-                "clause": "",
+                "clause": "that while I appreciate, and even enjoy, contact from readers, I'm absolutely not interested in whatever spammy commercial venture you're proposing",
                 "embedded clauses": []

```

Comment:
I would argue that "interested in whatever spammy commercial venture you're proposing" is an EC?

Reply:
I think that it's a free relative because it's not interrogative. I think actually that "whatever" is never used with an EC